src/Implementation/InputDefinition/ValueDefinition.php

```diff
@@ -2,11 +2,34 @@
 namespace De\Idrinth\SimpleConsole\Implementation\InputDefinition;
 
 use De\Idrinth\SimpleConsole\Implementation\InputDefinition;
+use InvalidArgumentException;
 
 class ValueDefinition extends InputDefinition
 {
-    public function __construct($name, $required, $regex, $default)
+    /**
+     * @param string $name
+     * @param boolean $required
+     * @param string $regex
+     * @param mixed $default
```

Comment:
null|mixed. mixed includes null too but its the default value so itll be better for reading i think